content/blog/2020/08/2020-08-24-custom-distribution-service.adoc

```diff
@@ -0,0 +1,234 @@
+---
+layout: post
+title: "Custom Distribution Service : Phase 3 Blogpost"
+tags:
+- service
+- distribution
+- cloud-native
+- gsoc
+- gsoc2020
+- packaging
+- platform-sig
+author: sladyn98
+opengraph:
+  image: /images/gsoc/2020/custom_distribution_service/cds_gsoc.png
+---
+
+
+Hello everyone,
+
+This is the final blog post for the Custom Distribution Service project during the Google Summer of Code timeline .I have mixed feelings since we are almost near the finish line for one of the most amazing open source programs out there. However it is time to wrap things up for this project and achieve a state where the project can be built upon and extended further. This phase has been super busy with respect to the bug fixes, testing and getting the project hosted, so let us get straight into the updates.
```

Comment:
```suggestion
This is the final blog post for the Custom Distribution Service project during the Google Summer of Code timeline.
I have mixed feelings since we are almost near the finish line for one of the most amazing open source programs out there.
However, it is time to wrap things up for this project and achieve a state where the project can be built upon and extended further.
This phase has been super busy with respect to the bug fixes, testing and getting the project hosted, so let us get straight into the phase 3 updates.
```